lib/control.js

```diff
@@ -98,8 +98,23 @@ const sequential = (fns, context, callback) => {
   next();
 };
 
+const runIf = (condition, defaultVal, asyncFn, ...args) => {
+  if (typeof defaultVal === 'function') {
+    args.unshift(asyncFn);
+    asyncFn = defaultVal;
+    defaultVal = undefined;
+  }
+  if (condition) {
+    asyncFn(...args);
+  } else {
+    const callback = common.last(args);
+    callback(null, defaultVal);
```

Comment:
Why not `common.safeCallback`?

Reply:
Forgot about this feature. Rewritten now.